shell/components/IconOrSvg.vue

```diff
@@ -24,8 +24,8 @@ const cssCache = {};
 
 const colors = {
   header: {
-    color: '--header-btn-text',
-    hover: '--header-btn-text-hover'
+    color: '--on-tertiary',
```

Comment:
fallbacks needed?

Reply:
This requires some effort.. I introduced a change to attempt this in the component.